build.gradle

```diff
@@ -57,7 +62,7 @@ subprojects {
 
 allprojects {
     dependencies {
-        implementation files("./component-locator-1.0.0.jar") // Points to local work in progress, will be replaced with released version when ready
```

Comment:
Change to implementation 1.0.3

Reply:
implementation "${locatorGroup}:${locatorModule}:1.0.3"
not from files but from artifactory

Reply:
implementation com.synopsys.integration:component-locator:1.0.3

Reply:
Please pull this branch and push a commit with the changes you are suggesting.

Reply:
This branch does not have ${locatorGroup} ?